Bixal/move.mil
# fixed indent issue causing invalid yaml syntax in an ebextension

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Fixes an indent issue that caused an invalid AMI to be built because of invalid YAML syntax

## Testing

To verify the changes proposed in this pull request…

1. Deploy the AMI that is built
2. Ensure that the deploy does not fail because of invalid YAML syntax
